main.js

```diff
@@ -0,0 +1,135 @@
+const dropdown = document.querySelector("select");
+dropdown.addEventListener("change", sortOnChange);
+document.getElementById('taskInput').addEventListener("keydown",insert);
+let storage = {};
+
+if(localStorage.getItem("storage") === null){
```

Comment:
Adding if statement to a function. Calling the function at the end of the functions declarations